btree/btree.c

```diff
@@ -2359,7 +2360,7 @@ static struct m0_btree_oimpl *alloc(int height)
 /**
  * Checks if given segaddr is within segment boundaries.
 */
-static int address_is_valid(struct segaddr addr)
+static int address_in_segment(struct segaddr addr)
```

Comment:
If we need to check if the address falls within segment boundaries then return bool with true or false indicating success or failure.

Reply:
okay